client/container.go

```diff
@@ -732,19 +612,21 @@ func (x *PrmAnnounceSpace) SetValues(vs []container.SizeEstimation) {
 //
 // Context is required and must not be nil. It is used for network communication.
 //
+// Announcements parameter SHOULD NOT be empty slice.
```

Comment:
```suggestion
// Announcements parameter MUST NOT be empty slice.
```
since we don't have any conditions when it can be empty

Reply:
done

Reply:
wow, reviewers [fight](https://github.com/nspcc-dev/neofs-sdk-go/pull/424#discussion_r1205471289). well... okay... but not agree